althea_kernel_interface/src/ip_addr.rs

```diff
@@ -44,6 +45,38 @@ impl dyn KernelInterface {
         }
     }
 
+    /// After receiving an ipv6 addr from the exit, this function adds that ip
+    /// to /etc/network/addr. SLAAC takes this ip and assigns a /64 to hosts that connect
+    /// to the router
+    pub fn setup_ipv6_slaac(&self, router_ipv6_str: IpNetwork) {
```

Comment:
this is very clean, does it disrupt the system when it's run?

Reply:
When running /etc/init.d/network reload it seemed like the router would hang for around half a second but would then resolve